restler/unit_tests/log_baseline_test_files/test_replay.txt

```diff
@@ -0,0 +1,7 @@
+-> PUT /B/B HTTP/1.1\r\nAccept: application/json\r\nHost: unittest\r\nContent-Type: application/json\r\nAuthorization: valid_unit_test_token\r\n\r\n

```

Comment:
Note: I did not expect to need to include `r\nAuthorization: valid_unit_test_token` but replay during the unit tests was failing without this.